decidim-core/config/initializers/dragonfly.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require "dragonfly"
+require "avatar_magick"
+
+# Configure
+Dragonfly.app.configure do
+  plugin :imagemagick
+  plugin :avatarmagick
+
+  secret "c7e2a7239bd280c596739e13aba675994af2d570ad9e366b6166d15c2f4db4d9"
+
+  url_format "/media/:job/:name"
+
+  datastore :file,
+            root_path: Rails.root.join("public/system/dragonfly", Rails.env),
+            server_root: Rails.root.join("public")
+end
+
+# Logger
+Dragonfly.logger = Rails.logger
+
+# Mount as middleware
+Rails.application.middleware.use Dragonfly::Middleware
+
+# Add model functionality
+if defined?(ActiveRecord::Base)
+  ActiveRecord::Base.extend Dragonfly::Model
```

Comment:
WUUUT nooooo monkeypatchingggg. Why can't you explicitly add this on the models that need this?